src/Client.php

```diff
@@ -95,10 +95,10 @@ public function getClientApi(): ClientApi
     public function getNode($key = null): Node
     {
         if (null === $key) {
-            $key = array_rand($this->nodes);
+            $key = \array_rand($this->nodes);
         }
 
-        if (!isset($this->nodes[$key])) {
+        if (empty($this->nodes[$key])) {
```

Comment:
Why is this change necessary?

Reply:
For some reasons, using the `empty` function to check whether the variable is set.
And I also think this will be readable than using the `isset` function.

You can see more details about [that](http://php.net/manual/en/function.empty.php) in official PHP doc.

Reply:
I think `isset` is pretty straight forward and readable, a common contruct to check if a key in an array exists. One could use `array_key_exists` to make it more readable, but in the end the array key has to exist and the related value should be of type `Node`: both checks are there (isset + return type). So the function works as intended and using empty will not change it to the better.

Reply:
Ok. It makes sense and I will revert this.

Reply:
It has been reverted.